core/src/main/scala/za/co/absa/spline/harvester/postprocessing/ViewAttributeAddingFilter.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.harvester.postprocessing
+
+import org.apache.commons.configuration.Configuration
+import za.co.absa.spline.harvester.HarvestingContext
+import za.co.absa.spline.producer.model.v1_1.{AttrOrExprRef, DataOperation, ExecutionPlan}
+
+class ViewAttributeAddingFilter(conf: Configuration) extends AbstractPostProcessingFilter {
+
+  override def processExecutionPlan(plan: ExecutionPlan, ctx: HarvestingContext): ExecutionPlan = {
+    val views = plan.operations.other
+      .map(_.filter(_.name.get.startsWith("View")))
+      .getOrElse(Seq.empty)
```

Comment:
Since you are unwrapping monad anyway in this code, by doing it first you can save on extra `map` and make it a bit easier to read:
```suggestion
    val views = plan.operations.other
      .getOrElse(Seq.empty)
      _.filter(_.name.get.startsWith("View"))
```